src/extras/liveshare.ts

```diff
@@ -0,0 +1,86 @@
+import * as vsls from 'vsls/vscode'
+import * as vscode from 'vscode'
+import { lw } from '../lw'
+import * as url from 'url'
+import * as utils from '../utils/utils'
+
+export class LiveShare {
```

Comment:
Current design of the extension avoid using classes to implement features. You may refer to other modules in the `extra\` folder to see the implementation. Generally we rely on `export` to expose functions to LW.